src/Subscription/ThrowableToErrorContextTransformer.php

```diff
@@ -56,6 +56,10 @@ private static function transformThrowable(Throwable $error): array
      */
     private static function transformTrace(array $trace): array
     {
+        if (array_key_exists('class', $trace) && is_string($trace['class'])) {
+            $trace['class'] = str_replace("\x00", '', $trace['class']);
```

Comment:
Any better suggestions @DavidBadura ?

Reply:
Shouldn't that be its own PR?

Reply:
Right now, this is not doing any harm. When we adjust the table type to jsonb Postgres checks if the json is valid and doesn't like null bytes in there.

I don't think that this must be in a separate bugfix PR but I could extract it. The only thing is if the user already has null bytes in the table. Then he would need to remove them. Or do you have a case in mind?